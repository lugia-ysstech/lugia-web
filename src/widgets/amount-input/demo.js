/**
 *
 * create by ligx
 *
 * @flow
 */
import * as React from 'react';
import Theme from '../theme';
import AmountInput from './index';
import Widget from '../consts/index';
import styled from 'styled-components';
import { getBoxShadow } from '@lugia/theme-utils';

class LimitAmountInput extends React.Component<any, any> {
  constructor(props: any) {
    super(props);
    this.state = { value: '' };
  }

  onChange = ({ newValue: value }: any) => {
    this.setState({ value });
    this.props.onChange({ newValue: value });
  };

  render() {
    return <AmountInput value={this.state.value} onChange={this.onChange} />;
  }
}
class DefaultValueAmountInput extends React.Component<any, any> {
  render() {
    return <AmountInput defaultValue={'123456'} onChange={this.props.onChange} />;
  }
}
const Wrapper = styled.div`
  float: left;
  margin-left: 50px;
`;
export default () => {
  const view = {
    [Widget.AmountInput]: {
      Container: {
        normal: {
          width: 400,
          height: 40,
          border: {
            top: 10,
          },
        },
      },
      AmountInputPrefix: { normal: { fontSize: 14, color: 'blue' } },
      AmountTip: {
        Container: {
          normal: {
            background: {
              color: '#eee',
            },
          },
        },
        TooltipTitle: {
          normal: {
            color: '#4d63ff',
            fontSize: 16,
          },
        },
      },
    },
  };
  const onChange = (cmpName: string) => (value: any) => {};
  return (
    <div>
      <Wrapper>
        <p>formatter AmountInput</p>
        <AmountInput placeholder={'请填写金额'} />
        <p>default AmountInput</p>
        <AmountInput placeholder={'请填写金额'} />
        <p>禁用状态 </p>
        <AmountInput size={'default'} disabled={true} />
      </Wrapper>
      <Wrapper>
        <Theme config={view}>
          <p>small size</p>
          <AmountInput size={'small'} placeholder={'请填写金额'} />
          <p>default size</p>
          <AmountInput placeholder={'请填写金额'} />
          <p>large size</p>
          <AmountInput size={'large'} placeholder={'请填写金额'} disabled={true} />
        </Theme>
      </Wrapper>
      <Wrapper>
        <p>受限Input</p>
        <LimitAmountInput onChange={onChange('limit')} />
        <p>有默认值的 受限Input</p>
        <DefaultValueAmountInput value={'123456'} onChange={onChange('limit')} />
      </Wrapper>
      <Wrapper>
        <p>amountPrefix: '¥' transform: false </p>
        <AmountInput amountPrefix="¥" transform={false} />
        <p>amountPrefix: '$' transform: false </p>
        <AmountInput amountPrefix="$" transform={false} />
      </Wrapper>
    </div>
  );
};
