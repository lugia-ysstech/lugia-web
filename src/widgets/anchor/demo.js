/**
 *
 * create by guorg
 *
 * @flow
 */
import React from 'react';
import styled from 'styled-components';
import Theme from '../theme';
import Widget from '../consts/index';
import Anchor from './index';

const { Link } = Anchor;
const DemoBox = styled.div`
  padding: 20px;
  margin-left: ${props => 120 + props.cur * 120}px;
  width: 209px;
  height: 203px;
`;

export default class DrawerDemo extends React.Component<any, any> {
  constructor() {
    super();
  }
  click = () => {
    const anchorElement = document.getElementById('anchor-link');
    if (anchorElement) {
      anchorElement.scrollIntoView();
    }
  };

  handleLinkClick = (e, href) => {
    if (href) {
      const name = href.slice(1);
      const anchorElement = document.getElementById(name);
      if (anchorElement) {
        anchorElement.scrollIntoView({ block: 'start', behavior: 'smooth' });
      }
    }
  };
  render() {
    return (
      <div>
        <DemoBox cur={0}>
          <Anchor slideType="circle" useHref={false} onClick={this.handleLinkClick}>
            <Link title="anchor-link1" href="#anchor-link1" />
            <Link title="anchor-link2" href="#anchor-link2" />
            <Link title="anchor-link3" href="#anchor-link3">
              <Link title="anchor-link3.1" href="#anchor-link3.1" />
              <Link title="anchor-link3.2" href="#anchor-link3.2" />
            </Link>
            <Link title="anchor-link4" href="#anchor-link4" />
          </Anchor>
        </DemoBox>
        <DemoBox cur={1}>
          <Anchor slideType="line">
            <Link title="anchor-link1" href="#anchor-link1" />
            <Link title="anchor-link2" href="#anchor-link2" />
            <Link title="anchor-link3" href="#anchor-link3">
              <Link title="anchor-link3.1" href="#anchor-link3.1" />
              <Link title="anchor-link3.2" href="#anchor-link3.2" />
            </Link>
            <Link title="anchor-link4" href="#anchor-link4" />
          </Anchor>
        </DemoBox>
        <DemoBox cur={2}>
          <Anchor slideType="none" affix={false}>
            <Link title="anchor-link1" href="#anchor-link1" />
            <Link title="anchor-link2" href="#anchor-link2" />
            <Link title="anchor-link3" href="#anchor-link3">
              <Link title="anchor-link3.1" href="#anchor-link3.1" />
              <Link title="anchor-link3.2" href="#anchor-link3.2" />
            </Link>
            <Link title="anchor-link4" href="#anchor-link4" />
          </Anchor>
        </DemoBox>
        <div style={{ marginTop: '100px' }} id="anchor-link1" name="anchor-link1">
          anchor-link1
        </div>
        <div style={{ marginTop: '300px' }} id="anchor-link2" name="anchor-link2">
          anchor-link2
        </div>
        <div style={{ marginTop: '300px' }} id="anchor-link3" name="anchor-link3">
          anchor-link3
        </div>
        <div style={{ marginTop: '200px' }} id="anchor-link3.1" name="anchor-link3.1">
          anchor-link3.1
        </div>
        <div style={{ marginTop: '200px' }} id="anchor-link3.2" name="anchor-link3.2">
          anchor-link3.2
        </div>
        <div style={{ marginTop: '300px' }} id="anchor-link4" name="anchor-link4">
          anchor-link4
        </div>
      </div>
    );
  }
}
