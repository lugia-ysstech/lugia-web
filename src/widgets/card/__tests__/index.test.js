/**
 *
 * create by ligx
 *
 * @flow
 */
import React from 'react';

import 'jest-styled-components';
import renderer from 'react-test-renderer';
import { CardDemo } from '../demo';

describe('Card', () => {
  it('css', () => {
    const Target = <CardDemo />;
    expect(renderer.create(Target).toJSON()).toMatchSnapshot();
  });
});
