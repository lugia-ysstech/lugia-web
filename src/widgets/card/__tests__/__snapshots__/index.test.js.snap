// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card css 1`] = `
.c10 {
  box-sizing: border-box;
  display: inline-block;
  text-align: center;
  white-space: nowrap;
  position: relative;
  vertical-align: middle;
  color: rgba(0,0,0,0.65);
  border-radius: 50%;
  line-height: 3.2rem;
  min-width: 2.4rem;
  min-height: 2.4rem;
}

.c12 {
  width: 100%;
  height: 100%;
  background-color: transparent;
}

.c11 {
  min-width: 2.4rem;
  min-height: 2.4rem;
  background-color: transparent;
}

.c1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-width: 5rem;
  min-height: 2rem;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c8 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-width: 5rem;
  min-height: 2rem;
  text-align: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c14 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-width: 5rem;
  min-height: 2rem;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 {
  font-size: 1.2rem;
  display: inline-block;
  width: 100%;
}

.c16 {
  width: 20rem;
  height: 16rem;
  display: inline-block;
  max-width: 100%;
  max-height: 100%;
}

.c18 {
  width: 100%;
  height: 100%;
  display: inline-block;
  max-width: 100%;
  max-height: 100%;
}

.c15 {
  position: relative;
  width: 20rem;
  height: 15rem;
  display: block;
}

.c17 {
  position: relative;
  width: 20rem;
  height: 100%;
  display: inline-block;
}

.c9 {
  position: relative;
  text-align: center;
  width: 16rem;
  height: 10rem;
  display: block;
}

.c13 {
  position: relative;
  text-align: center;
  width: 8rem;
  height: 100%;
  display: inline-block;
}

.c5 {
  text-align: inherit;
  display: inline-block;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  font-size: 1.6rem;
  width: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c3 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c4 {
  width: 0.5rem;
  height: 2rem;
  background-color: #4d63ff;
  border-top-left-radius: 0.5rem;
  border-top-right-radius: 0.5rem;
  border-bottom-right-radius: 0.5rem;
  border-bottom-left-radius: 0.5rem;
  margin-top: 1.3rem;
  margin-bottom: 0rem;
  margin-left: 0.5rem;
  margin-right: 0rem;
}

.c7 {
  width: 100%;
  height: 0.1rem;
  border-bottom-width: 0.1rem;
  border-bottom-style: solid;
  border-bottom-color: #e8e8e8;
  margin-top: 1rem;
  margin-bottom: 0rem;
  margin-left: 0rem;
  margin-right: 0rem;
}

.c6 {
  text-align: inherit;
  font-size: 1.4rem;
  color: #666666;
}

.c19 {
  font-size: 1.4rem;
  font-weight: 400;
  top: 1.6rem;
  right: 2rem;
  position: absolute;
}

.c27 {
  position: relative;
  cursor: pointer;
  white-space: nowrap;
  padding: 0 20px;
  display: inline-block;
  text-align: center;
}

.c25 {
  position: relative;
  cursor: pointer;
  white-space: nowrap;
  padding: 0 20px;
  display: inline-block;
  text-align: center;
}

.c25 > div::before {
  content: '';
  background: #4d63ff;
  border-radius: 2px;
  position: absolute;
  bottom: -1px;
  width: 100%;
  height: 2px;
  left: 50%;
  -webkit-animation: gaMprO 0.2s linear forwards;
  animation: gaMprO 0.2s linear forwards;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
}

.c26 {
  position: relative;
  display: inline-block;
  box-sizing: border-box;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  line-height: 4.2rem;
  height: 4.2rem;
}

.c26:focus {
  color: #4d63ff;
}

.c23 {
  width: 100%;
  white-space: nowrap;
  overflow: hidden;
  float: left;
}

.c24 {
  display: inline-block;
  box-sizing: border-box;
  white-space: nowrap;
  -webkit-transition: all 0.5s;
  transition: all 0.5s;
  min-width: 100%;
  -webkit-transform: translateX(0rem);
  -ms-transform: translateX(0rem);
  transform: translateX(0rem);
}

.c22 {
  position: relative;
  z-index: 99;
  overflow: hidden;
  text-align: left;
}

.c28 {
  position: relative;
  background: #fff;
  min-width: 100px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c29 {
  overflow: hidden;
  padding: 10px;
  width: 100%;
  height: 0;
  display: none;
  display: block;
  z-index: 10;
  height: 100%;
}

.c30 {
  overflow: hidden;
  padding: 10px;
  width: 100%;
  height: 0;
  display: none;
}

.c21 {
  -webkit-box-sizing: border-box;
  box-sizing: border-box;
  position: relative;
  min-height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 {
  margin-left: 50px;
  margin-top: 50px;
  display: inline-block;
}

.c20 {
  display: inline-block;
  margin-left: 20px;
}

.c31 {
  text-align: center;
  width: 100%;
  height: 100%;
}

.c32 {
  text-align: center;
  font-size: 16px;
  color: #666;
  height: 45px;
  padding-top: 34px;
  padding-bottom: 26px;
  width: 100%;
}

.c33 {
  text-align: center;
  color: #4d63ff;
  display: inline-block;
}

.c34 {
  position: absolute;
  text-align: center;
  font-size: 14px;
  border-radius: 20px;
  margin-top: 32px;
  color: #fff;
  bottom: 10%;
  width: 80%;
  background-color: #4d63ff;
  height: 32px;
  border: 2px solid #4d63ff;
  left: 10%;
}

<div
  className="c0"
>
  <p>
    基本样式
  </p>
  <p>
    tip样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
        }
      }
      type="tip"
    >
      <div
        className="c2"
        style={Object {}}
        type="tip"
      >
        <div
          className="c3"
          style={Object {}}
        >
          <div
            className="c4"
            style={Object {}}
          />
          <div
            className="c5"
            style={
              Object {
                "fontWeight": 700,
                "paddingBottom": "0rem",
                "paddingLeft": "1rem",
                "paddingRight": "1rem",
                "paddingTop": "1.3rem",
              }
            }
            type="tip"
          >
            this is title
          </div>
        </div>
        <div
          className="c6"
          style={
            Object {
              "paddingBottom": "0rem",
              "paddingLeft": "1.6rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          <div>
            this is description
          </div>
          <div>
            this is description
          </div>
           
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "20rem",
          "width": "50rem",
        }
      }
      type="tip"
    >
      <div
        className="c2"
        style={Object {}}
        type="tip"
      >
        <div
          className="c3"
          style={
            Object {
              "height": "1.8rem",
            }
          }
        >
          <div
            className="c4"
            style={
              Object {
                "borderBottomLeftRadius": "0rem",
                "borderTopLeftRadius": "0rem",
                "height": "1.8rem",
                "marginBottom": "0rem",
                "marginLeft": "0rem",
                "marginRight": "0rem",
                "marginTop": "1.3rem",
                "width": "0.4rem",
              }
            }
          />
          <div
            className="c5"
            style={
              Object {
                "fontWeight": 700,
                "paddingBottom": "0rem",
                "paddingLeft": "1rem",
                "paddingRight": "1rem",
                "paddingTop": "1.3rem",
              }
            }
            type="tip"
          >
            this is title
          </div>
        </div>
        <div
          className="c7"
          style={Object {}}
        />
        <div
          className="c6"
          style={
            Object {
              "paddingBottom": "0rem",
              "paddingLeft": "1.6rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          <div>
            this is description
          </div>
          <div>
            this is description
          </div>
           
        </div>
      </div>
    </div>
  </div>
  <p>
    头像样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c8"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "22rem",
          "width": "16rem",
        }
      }
      type="avatar"
    >
      <div
        className="c9"
        style={
          Object {
            "height": "10rem",
            "marginBottom": "0rem",
            "marginLeft": "0rem",
            "marginRight": "0rem",
            "marginTop": "2rem",
            "width": "16rem",
          }
        }
      >
        <div
          className="c10"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          <div
            className="c11"
            style={
              Object {
                "height": "6rem",
                "width": "6rem",
              }
            }
          >
            <img
              alt=""
              className="c12"
              src="http://192.168.102.73:8081/BigFrontend/Work/ued/lugia/raw/4d4bd6db04b1c6015acf4c933607956a9f2d62a1/lugiaweb%E7%BB%84%E4%BB%B6/%E5%8D%A1%E7%89%87/Bitmap2.png"
              style={
                Object {
                  "borderBottomLeftRadius": "50%",
                  "borderBottomRightRadius": "50%",
                  "borderTopLeftRadius": "50%",
                  "borderTopRightRadius": "50%",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="c2"
        style={Object {}}
        type="avatar"
      >
        <div
          className="c5"
          style={
            Object {
              "fontWeight": 500,
              "paddingBottom": "0rem",
              "paddingLeft": "0rem",
              "paddingRight": "1rem",
              "paddingTop": "1rem",
            }
          }
          type="avatar"
        >
          this is title
        </div>
        <div
          className="c6"
          style={
            Object {
              "paddingBottom": "0rem",
              "paddingLeft": "0rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          this is description
           
        </div>
      </div>
    </div>
  </div>
  <p>
    头像样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "14rem",
          "width": "26rem",
        }
      }
      type="avatar"
    >
      <div
        className="c13"
        style={
          Object {
            "marginBottom": "0rem",
            "marginLeft": "0rem",
            "marginRight": "0rem",
            "marginTop": "1rem",
            "width": "8rem",
          }
        }
      >
        <div
          className="c10"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          <div
            className="c11"
            style={
              Object {
                "height": "6rem",
                "width": "6rem",
              }
            }
          >
            <img
              alt=""
              className="c12"
              src="http://192.168.102.73:8081/BigFrontend/Work/ued/lugia/raw/4d4bd6db04b1c6015acf4c933607956a9f2d62a1/lugiaweb%E7%BB%84%E4%BB%B6/%E5%8D%A1%E7%89%87/Bitmap2.png"
              style={
                Object {
                  "borderBottomLeftRadius": "50%",
                  "borderBottomRightRadius": "50%",
                  "borderTopLeftRadius": "50%",
                  "borderTopRightRadius": "50%",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="c2"
        style={Object {}}
        type="avatar"
      >
        <div
          className="c5"
          style={
            Object {
              "fontWeight": 500,
              "marginBottom": "0rem",
              "marginLeft": "2rem",
              "marginRight": "0rem",
              "marginTop": "1rem",
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1rem",
            }
          }
          type="avatar"
        >
          this is title
        </div>
        <div
          className="c6"
          style={
            Object {
              "marginBottom": "0rem",
              "marginLeft": "2rem",
              "marginRight": "0rem",
              "marginTop": "0rem",
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          <div>
            <div>
              this is description
            </div>
          </div>
           
        </div>
      </div>
    </div>
  </div>
  <p>
    图片样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c14"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "30rem",
          "opacity": 0.8,
          "width": "20rem",
        }
      }
      type="image"
    >
      <div
        className="c15"
        style={
          Object {
            "height": "15rem",
            "width": "20rem",
          }
        }
      >
        <img
          className="c16"
          src="http://192.168.102.73:8081/BigFrontend/Work/ued/lugia/raw/2eac1a340185301d24d6fac426aebd9abe6dea0e/lugiaweb%E7%BB%84%E4%BB%B6/%E5%8D%A1%E7%89%87/18081548404150_.pic_hd.jpg"
          style={Object {}}
        />
      </div>
      <div
        className="c2"
        style={Object {}}
        type="image"
      >
        <div
          className="c5"
          style={
            Object {
              "color": "pink",
              "fontWeight": 500,
              "height": "2rem",
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1rem",
              "width": "10rem",
            }
          }
          type="image"
        >
          this is title
        </div>
        <div
          className="c6"
          style={
            Object {
              "color": "blue",
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          this is description
           
        </div>
      </div>
    </div>
  </div>
  <p>
    图片样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "15rem",
          "width": "36rem",
        }
      }
      type="image"
    >
      <div
        className="c17"
        style={
          Object {
            "width": "20rem",
          }
        }
      >
        <img
          className="c18"
          src="http://192.168.102.73:8081/BigFrontend/Work/ued/lugia/raw/2eac1a340185301d24d6fac426aebd9abe6dea0e/lugiaweb%E7%BB%84%E4%BB%B6/%E5%8D%A1%E7%89%87/18081548404150_.pic_hd.jpg"
          style={Object {}}
        />
      </div>
      <div
        className="c2"
        style={Object {}}
        type="image"
      >
        <div
          className="c5"
          style={
            Object {
              "fontWeight": 500,
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1rem",
            }
          }
          type="image"
        >
          this is title
        </div>
        <div
          className="c6"
          style={
            Object {
              "paddingBottom": "0rem",
              "paddingLeft": "1rem",
              "paddingRight": "1rem",
              "paddingTop": "1.4rem",
            }
          }
        >
          this is description
           
        </div>
      </div>
    </div>
  </div>
  <p>
    组合样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "24rem",
          "width": "40rem",
        }
      }
      type="combo"
    >
      <div
        className="c19"
        style={Object {}}
      >
        操作
      </div>
      <div
        className="c2"
        content={
          <ForwardRef>
            <ForwardRef
              data={
                Array [
                  Object {
                    "content": "content of Tab1",
                    "title": "Tab1",
                  },
                  Object {
                    "content": "content of Tab2",
                    "title": "Tab2",
                  },
                  Object {
                    "content": "content of Tab3",
                    "title": "Tab3",
                  },
                ]
              }
            />
          </ForwardRef>
        }
        style={Object {}}
        type="combo"
      >
        <div
          className="c20"
        >
          <div
            className="OutContainer  c21"
            style={Object {}}
          >
            <div
              className="c22"
              style={
                Object {
                  "backgroundColor": "#fff",
                }
              }
            >
              <div
                className="c23"
                style={Object {}}
              >
                <div
                  className="c24"
                  style={
                    Object {
                      "borderBottomColor": "#e8e8e8",
                      "borderBottomStyle": "solid",
                      "borderBottomWidth": "0.1rem",
                    }
                  }
                  x={-0}
                >
                  <div
                    className="c25"
                    disabled={false}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={
                      Object {
                        "color": "#4d63ff",
                      }
                    }
                  >
                    <div
                      className="lineTitle  c26"
                      disabled={false}
                      style={
                        Object {
                          "lineHeight": "3.4rem",
                        }
                      }
                    >
                      Tab1
                    </div>
                  </div>
                  <div
                    className="c27"
                    disabled={false}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={Object {}}
                  >
                    <div
                      className="lineTitle  c26"
                      disabled={false}
                      style={
                        Object {
                          "lineHeight": "3.4rem",
                        }
                      }
                    >
                      Tab2
                    </div>
                  </div>
                  <div
                    className="c27"
                    disabled={false}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={Object {}}
                  >
                    <div
                      className="lineTitle  c26"
                      disabled={false}
                      style={
                        Object {
                          "lineHeight": "3.4rem",
                        }
                      }
                    >
                      Tab3
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="c28"
              style={Object {}}
            >
              <div
                className="c29"
                style={Object {}}
              >
                <div
                  className=""
                  style={Object {}}
                >
                  content of Tab1
                </div>
              </div>
              <div
                className="c30"
                style={Object {}}
              >
                <div
                  className=""
                  style={Object {}}
                >
                  content of Tab2
                </div>
              </div>
              <div
                className="c30"
                style={Object {}}
              >
                <div
                  className=""
                  style={Object {}}
                >
                  content of Tab3
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <p>
    组合样式
  </p>
  <div
    className="c0"
  >
    <div
      className="c1"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomColor": "#cccccc",
          "borderBottomLeftRadius": "0.5rem",
          "borderBottomRightRadius": "0.5rem",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0.1rem",
          "borderLeftColor": "#cccccc",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0.1rem",
          "borderRightColor": "#cccccc",
          "borderRightStyle": "solid",
          "borderRightWidth": "0.1rem",
          "borderTopColor": "#cccccc",
          "borderTopLeftRadius": "0.5rem",
          "borderTopRightRadius": "0.5rem",
          "borderTopStyle": "solid",
          "borderTopWidth": "0.1rem",
          "boxShadow": "0px 0px 6px 0px rgba(0, 0, 50, 0.1)",
          "height": "22rem",
          "width": "20rem",
        }
      }
      type="combo"
    >
      <div
        className="c2"
        content={<AmountCard />}
        style={Object {}}
        type="combo"
      >
        <div
          className="c31"
        >
          <div
            className="c32"
          >
            价格
          </div>
          <div
            className="c33"
          >
            <div
              className="c33"
              style={
                Object {
                  "fontSize": 32,
                }
              }
            >
              9
            </div>
            <div
              className="c33"
              style={
                Object {
                  "fontSize": 18,
                }
              }
            >
              .88
            </div>
          </div>
          <button
            className="c34"
          >
            购买
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
